### ISSUE TITLE
[Menubar Calendar] Deprecated function call error

### ISSUE THREAD

**Issue by jiajames** [raycast/extensions#13930]
### Extension

https://raycast.com/koinzhang/menubar-calendar

### Raycast Version

1.80.0

### macOS Version

14.0

#### Error:
```
SwiftError: Error Domain=EKErrorDomain Code=61 "-requestAccessToEntityType:completion: has been deprecated-calling this method is no longer allowed. Instead, use -requestFullAccessToEventsWithCompletion:, -requestWriteOnlyAccessToEventsWithCompletion:, or -requestFullAccessToRemindersWithCompletion:." UserInfo={NSLocalizedDescription=-requestAccessToEntityType:completion: has been deprecated-calling this method is no longer allowed. Instead, use -requestFullAccessToEventsWithCompletion:, -requestWriteOnlyAccessToEventsWithCompletion:, or -requestFullAccessToRemindersWithCompletion:.}
    at ChildProcess.<anonymous> (/Users/jamesjia/.config/raycast/extensions/5be54b64-8da3-4dd2-a2cd-fe53909246a7/index.js:33:30097)
    at ChildProcess.emit (node:events:519:28)
    at ChildProcess.emit (node:domain:488:12)
    at ChildProcess._handle.onexit (node:internal/child_process:294:12)
```

### Steps To Reproduce

This happens when attempting to refresh.

**Comment by koinzhang** [comment]
Thanks for the feedback, I'll fix this

**Comment by koinzhang** [comment]
Fixed in #14693

**Comment by koinzhang** [comment]
@raycastbot close this issue
